utility/src/test/java/org/oppia/android/util/networking/NetworkConnectionUtilDebugImplTest.kt

```diff
@@ -0,0 +1,238 @@
+package org.oppia.android.util.networking
+
+import android.app.Application
+import android.content.Context
+import android.net.ConnectivityManager
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.google.common.truth.Truth.assertThat
+import dagger.BindsInstance
+import dagger.Component
+import dagger.Module
+import dagger.Provides
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.oppia.android.testing.robolectric.RobolectricModule
+import org.oppia.android.testing.time.FakeOppiaClockModule
+import org.oppia.android.util.logging.EnableConsoleLog
+import org.oppia.android.util.logging.EnableFileLog
+import org.oppia.android.util.logging.GlobalLogLevel
+import org.oppia.android.util.logging.LogLevel
+import org.oppia.android.util.networking.NetworkConnectionUtil.ConnectionStatus
+import org.robolectric.Shadows.shadowOf
+import org.robolectric.annotation.Config
+import org.robolectric.annotation.LooperMode
+import org.robolectric.shadows.ShadowNetworkInfo
+import javax.inject.Inject
+import javax.inject.Singleton
+
+/** Tests for [NetworkConnectionUtilDebugImpl]. */
+@RunWith(AndroidJUnit4::class)
+@LooperMode(LooperMode.Mode.PAUSED)
+@Config(manifest = Config.NONE)
+class NetworkConnectionUtilDebugImplTest {
+
+  private val NO_CONNECTION = -1
+
+  @Inject
+  lateinit var networkConnectionUtilDebugImpl: NetworkConnectionUtilDebugImpl
+
+  @Inject
+  lateinit var context: Context
+
+  @Before
+  fun setUp() {
+    setUpTestApplicationComponent()
+  }
+
+  private fun setUpTestApplicationComponent() {
+    DaggerNetworkConnectionUtilDebugImplTest_TestApplicationComponent.builder()
+      .setApplication(ApplicationProvider.getApplicationContext())
+      .build()
+      .inject(this)
+  }
+
+  @Test
+  fun testGetCurrentConnectionStatus_activeWifiConnection_noForcedConnection_returnsWifi() {
```

Comment:
```suggestion
  fun testGetCurrentConnectionStatus_activeWifi_noForcedConnection_returnsWifi() {
```

Try to be consistent in naming. If you've used activeWifi in 3/4 tests then try to use the same in the other test as well. Ditto elsewhere.

Reply:
Done